NineChronicles.Headless.Tests/GraphTypes/StandaloneQueryTest.cs

```diff
@@ -555,6 +556,81 @@ public async Task GetTx()
             Assert.Equal(transaction.Actions.First().PlainValue.Inspection, plainValue);
         }
 
+        [Fact]
+        public async Task Products()
+        {
+            var userPrivateKey = new PrivateKey();
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
```

Comment:
It seems useless to set `StandaloneContext.NineChroniclesNodeService ` in this scope.

Reply:
this is need to be reset states

Reply:
As you said, `MakeMineChroniclesNodeService` was used to reset states. But what I wanted to say was that the `products` query doesn't use `StandaloneContext.NineChroniclesStandaloneService`.

Reply:
As a comment after I see `GraphQLTestBase`, `GraphQLTestBase` already has been initialized its `StandaloneContextFx.BlockChain` itself in its constructor. So it needs refactored its fixture logics and tests after.